GitUI/Translation/English.xlf

```diff
@@ -9585,7 +9585,8 @@ Selected Branch:
 {sAuthorDate}
 {sCommitDate}
 
-Current Branch:
+On currently checked out revision:
+{cHead} (checked out branch name or checked out commit hash)
```

Comment:
In the light of the recent discussions about scripts, I think we should consider renaming all the scripts' variables giving those human-readable names. E.g., `currentHead` for this one. 
I'm not requesting a change in the context of this PR, just raising a concern.